api/grpcserver.go

```diff
@@ -49,12 +49,18 @@ import (
 	"github.com/iotexproject/iotex-core/pkg/tracer"
 )
 
-// GRPCServer contains grpc server and the pointer to api coreservice
-type GRPCServer struct {
-	port        string
-	grpcServer  *grpc.Server
-	coreService CoreService
-}
+type (
+	// GRPCServer contains grpc server
+	GRPCServer struct {
+		port string
+		svr  *grpc.Server
+	}
+
+	// GRPCHandler contains the pointer to api coreservice
+	GRPCHandler struct {
```

Comment:
private struct and move to a new file

Reply:
Moving to a new file is a good idea, but files under `/api` are overflowing.
Current `grpcserver` is ok, compared with separating it into `grpcserver` and `grpchandler`